docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       - "3000:3000"
       - "6060"
     env_file:
-      - .env
+      - local.env
```

Comment:
Maybe leave the .env here to not break existing local setups?

Reply:
The problem with using the `.env` here is that the values would be different for local setups than for what gets deployed via codeship.  E.g., we need the UIURL to be wecarry.local:5001 locally, but not in the .env.encrypted file.

Reply:
Can we just tell everyone to copy their `.env` file to `local.env` ?

Reply:
Yes, I was just thinking of a way to avoid the trouble. When things like this change, it's often weeks later that someone hits a strange problem and doesn't know how to solve it. I guess in this case, the error message would be pretty clear, so it's fine as you have it.